arch/arm/src/rtl8720c/ameba_nvic.c

```diff
@@ -316,7 +316,7 @@ void up_irqinitialize(void)
 
   /* Restore the NVIC vector location to local */
 
-  memcpy(&__vectors, (void *) * (volatile uint32_t *)(NVIC_VECTAB)
+  memcpy(__vectors, (void *) * (volatile uint32_t *)(NVIC_VECTAB)
```

Comment:
seems that `sizeof(__vectors)` is now 0

Reply:
No, __vectors is an array defined at line 63:
```
static int (* __vectors[NR_IRQS - NVIC_IRQ_FIRST])(void);
```
not a link symbol.